models/cookies-preference.go

```diff
@@ -0,0 +1,9 @@
+package models
```

Comment:
Core page models to live in the renderer library, with service-specific models being defined in the respective frontend apps